lib/utils/KalturaFFMpegCmdGenerator.js

```diff
@@ -0,0 +1,23 @@
+
+const util = require('util');
+const KalturaFFMpegCmdResponse = require('./KalturaFFMpegCmdResponse');
+
+/**
+ * return a command line that should be used by the transcoding engine to transcode the ad
+ */
+class KalturaFFMpegCmdGenerator
+{
+
+    static generateCommandLineFormat(flavorId ,MediaInfoJson , apiConnector )
+    {
+        return apiConnector.handleRequset('flavorAsset','serveAdStitchCmd' ,[flavorId , MediaInfoJson]);
+    }
+
+    static fillCmdLineFormat(format ,sourcePath , outputPath)

```

Comment:
What does this command do -  it is not self explained by title - add docs before it